src/NumberGuessingGame.jsx

```diff
@@ -13,60 +13,38 @@ function getRandomNumber() {
 
 const MAX_ATTEMPTS = 5;
 
-class NumberGuessingGame extends Component {
-  constructor(props) {
-    super(props);
+const NumberGuessingGame = () => {
+  const [numberToGuess] = useState(getRandomNumber());
+  const [numberOfGuesses, setNumberOfGuesses] = useState(0);
+  const [latestGuess, setLatestGuess] = useState(null);
 
-    this.state = {
-      numberToGuess: getRandomNumber(),
-      numberOfGuesses: 0,
-      latestGuess: null,
-    };
-
-    /**
-     * These lines are required to make the methods/functions declared on this
-     *  class have the correct `this` object when they run.
-     */
-    this.handleGuess = this.handleGuess.bind(this);
-    this.handleReset = this.handleReset.bind(this);
-  }
-
-  handleGuess(guess) {
-    this.setState({
-      latestGuess: guess,
-      numberOfGuesses: this.state.numberOfGuesses + 1,
-    });
-  }
-
-  handleReset() {
-    this.setState({
-      numberToGuess: getRandomNumber(),
-      numberOfGuesses: 0,
-      latestGuess: null,
-    });
-  }
+const handleGuess = (guess) => {
+  setLatestGuess(Number(guess))
+  setNumberOfGuesses((prevGuesses) => prevGuesses + 1);
+}
 
-  render() {
-    const isCorrectGuess = this.state.latestGuess === this.state.numberToGuess;
+const handleReset = () => {
+  setNumberOfGuesses(0);
+  setLatestGuess(null);
```

Comment:
You have an incomplete function here. 
It's missing setting `numberToGuess: getRandomNumber()` and the closing braces.

Reply:
It's probably the extra brace on line 52